README.md

```diff
@@ -5,3 +5,7 @@ This repository contains training material for teaching purposes.
 [SETUP.md](./SETUP.md) helps you to setup a Vault training environment.
 
 Follow along with the [exercises](./exercises) afterwards.
+
+To complete the entire course (setup including all exercises) it will take you
```

Comment:
This might be helpful, so you know in advanced that you need a day to go through all the material